bin/pasta_form_patchwork_relations.py

```diff
@@ -0,0 +1,62 @@
+import sys
```

Comment:
Missing copyright header

Reply:
Yes, I was a bit confused as to the content of the header. Should I just replace your name with mine in the other headers. Do I also need to change `OTH Regensburg`, if yes to what? The name of my college?

Reply:
You don't do it on behalf of your college. In your case, i'd say to simply use your own name as copyright holder.